src/main/resources/application.yml

```
@@ -13,3 +12,6 @@ springdoc:
   swagger-ui:
     with-credentials: true
     path: /swagger
+
+test:
+  value: ${TEST_VALUE}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)